.codacy.yml

```diff
@@ -0,0 +1,20 @@
+---
+engines:
+  phpcs:
+    enabled: true
+    php_version: 7.3
+  eslint:
+    enabled: true
+    exclude_paths:
+      - '**/*.svg'
```

Comment:
Let's also disable remark-lint here, as was added to the Drupal PR: https://github.com/forumone/gesso/pull/441/files#diff-6e27af35533eda2e3e82d6d7f8654c26824396797716d37ac7df096a1b090503R11-R12

Reply:
👍🏼 thanks I must have missed that change.

Reply:
This has been done and should be good to go now.